docs/resources/policy_v2.md

```diff
@@ -0,0 +1,156 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cyral_policy_v2 Resource - terraform-provider-cyral"
+subcategory: ""
+description: |-
+    This resource allows management of various types of policies in the Cyral platform. Policies can be used to define access controls, data governance rules, and approval mechanisms to ensure compliance and security within your database environment.
+---
+
+# cyral_policy_v2 (Resource)
```

Comment:
Again, I would not use the term _v2 here. Let's think of a more suitable name.

Reply:
Let's keep the `_v2` and explained in my first comment.